src/Misc/layoutbin/powershell/Add-JavaCapabilities.ps1

```diff
@@ -63,6 +86,13 @@ if  (-not (Test-Path env:DISABLE_JAVA_CAPABILITY_HIGHER_THAN_9)) {
     }
 }
 
+# AdoptOpenJDK section
+$null = Add-CapabilityFromRegistryWithLastVersionAvailable -PrefixName 'jdk_' -PostfixName '_x64' -Hive 'LocalMachine' -View 'Registry64' -KeyName $jdk9AndGreaterNameAdoptOpenJDK -ValueName 'Path' -Value ([ref]$latestJre) -VersionSubdirectory $jvmHotSpot -MinimumMajorVersion $minimumMajorVersion9
```

Comment:
Could we leave priority for previous logic - to avoid unexpected breaking changes? (for example, for several jdks being installed there will be switch to another jdk after agent update)

Reply:
Yes, I think it is possible

Reply:
Updated logic, now it will check new reg keys, only if nothing is found in JavaSoft reg key.